pytorch_lightning/utilities/cloud_io.py

```diff
@@ -42,6 +41,18 @@ def get_filesystem(path: pathlike):
         return fsspec.filesystem("file")
 
 
+def move_to_cpu(item):
+    if isinstance(item, dict):
+        new_dict = item.copy()
+        for key, val in item.items():
+            new_dict[key] = move_to_cpu(val)
+        return new_dict
+    if isinstance(item, list):
+        return list(map(move_to_cpu, item))
+    if isinstance(item, torch.Tensor):
+        return item.cpu()
+
```

Comment:
check out `move_data_to_device` here: https://github.com/PyTorchLightning/pytorch-lightning/blob/c7511bcbb49565c5b24f368fa25862f740f3ec28/pytorch_lightning/utilities/apply_func.py#L92

Reply:
aye thanks! Didn't know about this one 👍